dev_guide/deployments.adoc

```diff
@@ -244,43 +260,50 @@ spec:
 
 == Viewing Deployment Logs
 
-To view the logs of the latest deployment for a given deployment configuration:
+To stream the logs of the latest revision for a given deployment configuration:
 
 ----
-$ oc logs dc/<deployment_config> [--follow]
+$ oc logs -f dc/<name>
 ----
 
-Logs can be retrieved either while the deployment is running or if it has
-failed. If the deployment was successful, there will be no logs to view.
+If the latest revision is running or failed, `oc logs` will return the logs of the
+process that is responsible for deploying your pods. If it is successful, `oc logs`
+will return the logs from a pod of your application.
+
+You can also view logs from older failed deployments, if and only if the old deployment
+exists and has not been pruned or deleted manually:
 
-You can also view logs from older deployments:
+----
+$ oc logs --version=1 dc/<name>
+----
+
+For more options on retrieving logs see:
 
 ----
-$ oc logs --version=1 dc/<deployment_config>
+$ oc logs -h
```

Comment:
Again not mandatory, but in docs the long form is usually more self-descriptive.